drivers/lds01/lds01.c

```diff
@@ -0,0 +1,156 @@
+#include "lds01.h"
```

Comment:
Global remark on this file - Using #define constants for LDS01 specifics values (number of value in one frame, length of frame, etc) could be great ease the read.

Reply:
Done